idearium-lib/test/query.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
```

Comment:
`getOptions` is an internal function function to that file and isn't exported. I have added more tests to cover all the different options though.